kairon/shared/channels/broadcast/whatsapp.py

```
@@ -360,17 +360,27 @@ def resend_broadcast(self):
+        status_count_key = f"retry_count_{retry_count}_status"
+        MessageBroadcastProcessor.add_event_log(
+            self.bot, MessageBroadcastLogType.common.value, self.reference_id, **{status_count_key: EVENT_STATUS.INPROGRESS.value}
+        )
 
-        _, non_sent_recipients = self.initiate_broadcast(message_list, is_resend=True)
-        failure_cnt = len(non_sent_recipients)
+        try :
+            _, non_sent_recipients = self.initiate_broadcast(message_list, is_resend=True)
+            failure_cnt = len(non_sent_recipients)
+            status = EVENT_STATUS.COMPLETED.value
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)